nautobot_netbox_importer/diffsync/models/extras.py

```diff
@@ -120,6 +120,8 @@ class CustomField(NautobotBaseModel):
     # Instead we store it in "actual_required" and fix it up only afterwards.
     actual_required: Optional[bool]
 
+    _ignored_fields = {"choices"}
```

Comment:
do we need to include `id` here as well?

Reply:
ya, that should combine with the base class